block_server.py

```diff
@@ -737,27 +596,19 @@ def delete_pv_from_db(self, name):
             del manager.pvs[self.port][name]
             del manager.pvf[fullname]
             del self.pvDB[name]
-            del PVDB[name]
 
     def add_string_pv_to_db(self, name, count=1000):
         # Check name not already in PVDB and that a PV does not already exist
-        if name not in PVDB and name not in manager.pvs[self.port]:
+        if name not in manager.pvs[self.port]:
             try:
-                print_and_log("Adding PV %s" % name)
-                new_pv = { name : {
-                    'type': 'char',
-                    'count': count,
-                    'value': [0],
-                    }
-                }
+                print_and_log("Adding PV {}".format(name))
+                new_pv = {name: char_waveform(count)}
                 self._cas.createPV(BLOCKSERVER_PREFIX, new_pv)
-                PVDB[name] = new_pv
```

Comment:
This looks like an important line but I am not sure. You probably looked at what it was doing, can I talk with you about it?

Reply:
Renamed PVDB to make it clearer that this is ok